Makefile

```diff
@@ -4,14 +4,20 @@
 
 .PHONY: clean default docs help lint
 
+# Set these in the environment to override them. This is helpful for
+# development if you have file ownership problems because the user
+# in the container doesn't match the user on your host.
+SOCORRO_UID ?= 10001
+SOCORRO_GID ?= 10001
```

Comment:
Because we're always passing `--build-arg` in `make build` and that rule gets used everywhere, it's best to have the defaults here.

However, this duplicates the same default values in the `Dockerfile`. I'm not sure what to do about that. We should never change these values since they're ops-approved. Maybe we can just leave it like this?